src/utils/cache-file.ts

```diff
@@ -0,0 +1,38 @@
+import { join } from 'path';
+import { createHash } from 'crypto';
+import { outputFileSync } from 'fs-extra';
+import { memoize } from 'lodash';
+
+// to use the same compiled regexp object all the time
+// See https://jsperf.com/string-match-str-re/1
+const memoizedRegexp = memoize(RegExp);
+const cachedTest = (
+  reString: string | undefined | null,
+  subject: string,
+): boolean => {
+  if (reString == null) {
+    return false;
+  }
+  return memoizedRegexp(reString).test(subject);
+};
+
+const cachedMd5 = memoize((path: string) =>
+  createHash('md5')
+    .update(path)
+    .digest('hex'),
+);
+
+// FIXME: I think there is testRegexp and testMatch in Jest configugation
+// There is a default setting for that, but maybe it's defaulted from scratch
+export function cacheFile(
```

Comment:
by default, `jest` provides default values for both [`testRegex` ](https://jestjs.io/docs/en/configuration#testregex-string)and [`testMatch`](https://jestjs.io/docs/en/configuration#testregex-string) and it doesn't have a default setting to say which one of them is the default setting to match test files because only one of them can be used at a time.

Reply:
Yup, I put that `FIXME` so that we can handle this in a future PR (implementation of that function hasn't changed). And I left `I think` to be sure the fixer will double check the input to fix it correctly and not ensure my comment was 100% right :-p